src/Polly.Hosting/Polly.Hosting.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net7.0;net6.0;netstandard2.0;net472;net462</TargetFrameworks>
+    <AssemblyTitle>Polly.Hosting</AssemblyTitle>
```

Comment:
`Polly.Hosting` doesn't sound right => `Polly.Extensions`?
We could then follow the familiar pattern with `Polly.Extensions.DependencyInjection`